plugins/inputs/docker/docker_test.go

```diff
@@ -247,8 +247,9 @@ func testStats() *types.StatsJSON {
 func TestDockerGatherInfo(t *testing.T) {
 	var acc testutil.Accumulator
 	d := Docker{
-		client:  nil,
-		testing: true,
+		client:            nil,
+		testing:           true,
+		GatherEnvironment: []string{"ENVVAR1", "ENVVAR2"},
```

Comment:
Can you add tests for envvars that are not found in the container, envvars with zero length values, multiple '=' signs, and any other edge cases you can think of.